packages/core/contracts/Escrow.sol

```diff
@@ -150,11 +164,16 @@ contract Escrow is IEscrow, ReentrancyGuard {
         manifestUrl = _url;
         manifestHash = _hash;
         status = EscrowStatuses.Pending;
+
+        remainingFunds = getBalance();
+        require(remainingFunds > 0, 'Escrow balance is zero');
+
         emit Pending(manifestUrl, manifestHash);
+        emit Fund(remainingFunds);
     }
 
     function abort() external override trusted notComplete notPaid {
```

Comment:
Is it actually worth to keep abort? When would this be called since we already have cancel function?